<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;SHORT NARUTO - PRETO&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;R$ 70,00&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_add_short_rosa_onep">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>540</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>ADICIONAR AO CARRINHO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>340</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/produtos/shorts/short-template-rosa-simpsons.png);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_carrinho">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/imgicons/icones/carr.png);

color: rgb(255, 255, 255);
border-style:outset;
border-width: 2px;
border-color:black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_add_short_preto_naruto">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>ADICIONAR AO CARRINHO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_add_short_rosa_simpsons">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>540</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>ADICIONAR AO CARRINHO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>6</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;SHORTS&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>161</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/produtos/shorts/short-template-preto-onep-remove.png);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_add_short_preto_onep">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>ADICIONAR AO CARRINHO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/produtos/shorts/short-template-rosa-naruto.png);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>60</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/produtos/shorts/short-template-preto-simpsons.png);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>161</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/produtos/shorts/short-template-rosa-onep.png);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;SHORT ONE PIECE - PRETO&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;R$ 70,00&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>210</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;SHORT SIMPSONS - PRETO&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;R$ 70,00&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icons/icones/Gamepad(1).png);
image: url(:/icons/icones/Gamepad (3).png);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_add_short_preto_simpsons">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>260</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>ADICIONAR AO CARRINHO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_voltar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>VOLTAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_add_short_rosa_naruto">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>540</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>ADICIONAR AO CARRINHO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/produtos/shorts/short-template-preto-naruto.png);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>490</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;SHORT ONE PIECE - ROSA&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;R$ 70,00&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>490</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;SHORT NARUTO - ROSA&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;R$ 70,00&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>490</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;SHORT SIMPSONS - ROSA&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;R$ 70,00&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tamanho_short_preto_naruto">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Display Semibold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(173, 173, 173);
border-bottom-color: rgb(173, 173, 173);</string>
   </property>
   <item>
    <property name="text">
     <string>P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="tamanho_short_preto_onepiece">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Display Semibold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(173, 173, 173);
border-bottom-color: rgb(173, 173, 173);</string>
   </property>
   <item>
    <property name="text">
     <string>P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="tamanho_short_preto_simpsons">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>230</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Display Semibold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(173, 173, 173);
border-bottom-color: rgb(173, 173, 173);</string>
   </property>
   <item>
    <property name="text">
     <string>P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="tamanho_short_rosa_naruto">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>510</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Display Semibold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(173, 173, 173);
border-bottom-color: rgb(173, 173, 173);</string>
   </property>
   <item>
    <property name="text">
     <string>P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="tamanho_short_rosa_onepiece">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>510</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Display Semibold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(173, 173, 173);
border-bottom-color: rgb(173, 173, 173);</string>
   </property>
   <item>
    <property name="text">
     <string>P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="tamanho_short_rosa_simpsons">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>510</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Display Semibold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(173, 173, 173);
border-bottom-color: rgb(173, 173, 173);</string>
   </property>
   <item>
    <property name="text">
     <string>P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="../produtos/produtos.qrc"/>
  <include location="img-icons.qrc"/>
 </resources>
 <connections/>
</ui>
